packages/svelteui-core/src/components/Chip/Chip.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import useStyles from './Chip.styles.js';
+	import { randomID } from '$lib/styles';
+	import { get_current_component } from 'svelte/internal';
+	import { createEventForwarder, useActions } from '$lib/internal';
+	import Box from '../Box/Box.svelte';
+	import type { ChipProps as $$ChipProps } from './Chip.styles.js';
+
+	interface $$Props extends $$ChipProps {}
+
+	export let use: $$Props['use'] = [],
+		element: $$Props['element'] = undefined,
+		className: $$Props['className'] = '',
+		override: $$Props['override'] = {},
+		color: $$Props['color'] = 'blue',
+		id: $$Props['id'] = randomID(),
+		disabled: $$Props['disabled'] = false,
+		value: $$Props['value'] = null,
+		checked: $$Props['checked'] = false,
+		label: $$Props['label'] = null,
+		radius: $$Props['radius'] = 'sm',
+    size: $$Props['size'] = 'md',
+    variant: $$Props['variant'] = 'outline',
+    // type: $$Props['type'] = 'checkbox',
```

Comment:
To remove, or missing feature?

Reply:
There is an issue with having a dynamic type of input while binding to the value of the input. Need to check how this is done else where (if its done elsewhere). Are you aware of any other components that have a dynamic input type?

What this is supposed to allow is to be able to choose between only allowing a single chip in a group to be selected (type radio buttons), or allowing multiple to be selected (type checkboxes).

Reply:
Hum, we currently don't have that behaviour.  Can you please tell me what issue are you having?

Reply:
Minimal reproducable example:
```svelte
<script>
let type = 'checkbox'
let checked = false
</script>

<input bind:type bind:value />
```
Gives this error: 
```
'checked' binding can only be used with <input type="checkbox">
```

So I don't think the input type can be dynamically changed between `checkbox` and `radio`. I think it should always be of type `checkbox` and then we just implement logic to allow or deny selecting multiple at the same time in a `ChipGroup`. Does that sound reasonable to you?

Reply:
We can go with that, another possibility is having two inputs with a conditional logic, since its only two, so it does not get out of hand. Would that also work?

```
{#if type === 'radio'}
 <input type="radio" bind:value />
{:else if type === 'checkbox'}
  <input type="checkbox" bind:value />
{/if}
```